### PR TITLE
daemon: remove kviator

### PR DESCRIPTION
**Pull request by leseb**
As per discussion here: https://github.com/ceph/ceph-docker/issues/497

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test ceph_ansible-kraken-centos7_cluster

**Comment by leseb** [comment]
jenkins test ceph_ansible-kraken-centos7_cluster

**Comment by leseb** [comment]
jenkins test ceph_ansible-kraken-centos7_cluster

File: ceph-releases/kraken/ubuntu/16.04/daemon/config.kv.etcd.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    until confd -onetime -backend ${KV_TYPE} -node ${CONFD_NODE_SCHEMA}${KV_IP}:${KV_PORT} ${CONFD_KV_TLS} -prefix="/${CLUSTER_PATH}/"; do
+      log "Waiting for confd to write initial templates..."
+      sleep 1
+    done
+
+    log "Creating Keyrings."
+    ceph-authtool /etc/ceph/${CLUSTER}.client.admin.keyring --create-keyring --gen-key -n client.admin --set-uid=0 --cap mon 'allow *' --cap osd 'allow *' --cap mds 'allow'
+    ceph-authtool /etc/ceph/${CLUSTER}.mon.keyring --create-keyring --gen-key -n mon. --cap mon 'allow *'
+    ceph-authtool /var/lib/ceph/bootstrap-osd/${CLUSTER}.keyring --create-keyring --gen-key -n client.bootstrap-osd --cap mon 'allow profile bootstrap-osd'
```
Comment (ErwanAliasr1) [inline ceph-releases/kraken/ubuntu/16.04/daemon/config.kv.etcd.sh]: for component in osd mds rgw; do
   ceph-authtool /var/lib/ceph/bootstrap-$component/${CLUSTER}.keyring --create-keyring --gen-key -n client.bootstrap-$component --cap mon 'allow profile bootstrap-$component' 
done

File: ceph-releases/kraken/ubuntu/16.04/daemon/variables_entrypoint.sh [hunk 2]
```diff
@@ -39,12 +40,19 @@
...
-  CONFD_KV_TLS="-scheme=https -client-ca-keys=${KV_CA_CERT} -client-cert=${KV_CLIENT_CERT} -client-key=${KV_CLIENT_KEY}"
-fi
-
 CEPH_OPTS="--cluster ${CLUSTER}"
 MOUNT_OPTS="-t xfs -o noatime,inode64"
+ETCDCTL_OPT="--peers ${KV_IP}:${KV_PORT}"
+
+# make sure etcd uses http or https as a prefix
+if [[ "$KV_TYPE" == "etcd" ]]; then
+  if [ ! -z "${KV_CA_CERT}" ]; then
```
Comment (ErwanAliasr1) [inline ceph-releases/kraken/ubuntu/16.04/daemon/variables_entrypoint.sh]: if [ -n

**Comment by ErwanAliasr1** [comment]
I don't really like a git commit message which only points to #something. A commit should be self contained. So please, just explain why you need a change in that commit.

**Comment by ErwanAliasr1** [comment]
typo on commit title s/disto/distro/

**Pull request merged**